modules/static/protocolbuffers/gofeatures/rsync.incl

```diff
@@ -0,0 +1,4 @@
++ LICENSE
++ google/protobuf/go_features.proto
++ */
```

Comment:
What are these stars for?

Reply:
I believe it is to recurse in subdirectories but not positive. I can try removing it to see if it still works.

Reply:
Removing that causes the fetch script to fail with `Failure: "buf.build/protocolbuffers/gofeatures" had no .proto files`.

Reply:
What does the rsync manual say?

Reply:
```

       Note that, when using the --recursive (-r) option (which is implied by -a), every subcomponent of every path is visited from the top down, so include/exclude patterns get applied recursively to each subcomponent's
       full name (e.g. to include "/foo/bar/baz" the subcomponents "/foo" and "/foo/bar" must not be excluded).  The exclude patterns actually short-circuit the directory traversal stage when rsync finds the files to send.
       If a pattern excludes a particular parent directory, it can render a deeper include pattern ineffectual because rsync did not descend through that excluded section of the hierarchy.  This is particularly important
       when using a trailing '*' rule.  For instance, this won't work:

              + /some/path/this-file-will-not-be-found
              + /file-is-included
              - *

       This fails because the parent directory "some" is excluded by the '*' rule, so rsync never visits any of the files in the "some" or "some/path" directories.  One solution is to ask for all directories in the
       hierarchy to be included by using a single rule: "+ */" (put it somewhere before the "- *" rule), and perhaps use the --prune-empty-dirs option.  Another solution is to add specific include rules for all the parent
       dirs that need to be visited.  For instance, this set of rules works fine:

              + /some/
              + /some/path/
              + /some/path/this-file-is-found
              + /file-also-included
              - *
```

Specifically:

> One solution is to ask for all directories in the hierarchy to be included by using a single rule: "+ */" (put it somewhere before the "- *" rule)

Reply:
Can you translate that - is there a tighter way to do this, or why or why not does this guarantee we are just limited to the two files we want?

Reply:
If we wanted to be the most strict, then we could add these lines:

```
google/
google/protobuf/
```

However, using `*/` is a shortcut to not have to include those, since we're using the `--prune-empty-dirs` rsync option.